bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java

```diff
@@ -541,9 +541,11 @@ protected void updateChannel(ChannelUID channelUID) {
             } catch (CommunicationException e) {
                 logger.debug("API communication problem while trying to update! thing={}, haId={}, error={}",
                         getThingLabel(), getThingHaId(), e.getMessage());
+                updateState(channelUID, UnDefType.UNDEF);
```

Comment:
Shouldn't you set the Thing status to OFFLINE here instead?

Reply:
This is a good remark ;) @bruestel : can you help me to explain why we don't do that ? I think there are cases where the API will trigger a CommunicationException but this is not a real problem and limited to a particular API call.

Reply:
We don't want to set the thing OFFLINE because it can happen in particular/temporary circunstances that the API will return an "error" code leading to this CommunicationException exception.
This was confirmed by @bruestel

Reply:
But then I wouldn't set the state to UNDEF, but simply leave the one that was set before.
UNDEF essentially means it has a state, which cannot be properly mapped to openHAB...

Reply:
Ok, reverted.